Content.Server/GameObjects/Components/GUI/ServerHandsComponent.cs

```diff
@@ -469,6 +480,42 @@ public bool ThrowItem()
             return false;
         }
 
+        public void StartPulling(PullableComponent pullable)
+        {
+            if (IsPulling)
+            {
+                if (PulledObject!.TryGetController(out PullController oldController))
+                {
+                    oldController.StopPull();
+                }
+
+                if (PulledObject.Owner.TryGetComponent(out ServerStatusEffectsComponent oldStatus))
+                {
+                    oldStatus.ChangeStatusEffectIcon(StatusEffect.Pulled, "/Textures/Interface/StatusEffects/Buckle/unbuckled.png");
```

Comment:
I don't think I need to say anything

Reply:
:eyes:
![image](https://user-images.githubusercontent.com/10968691/88399250-e35d9080-cdc6-11ea-9aa1-cee44239e04c.png)

Changed.